provisioning/security/tpm-provider-emulator/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/hsm/SecurityProviderTPMEmulator.java

```diff
@@ -66,8 +68,6 @@ public SecurityProviderTPMEmulator() throws SecurityProviderException
     {
         //SRS_SecurityProviderTPMEmulator_25_001: [ The constructor shall start the local TPM Simulator, clear persistent for EK and SRK if it exist, create persistent primary for EK and SRK. ]
         tpm = TpmFactory.localTpmSimulator();
-        clearPersistent(tpm, EK_PERSISTENT_HANDLE, "EK");
-        clearPersistent(tpm, SRK_PERSISTENT_HANDLE, "SRK");
```

Comment:
q - is clearing the ek and srk ever relevant for a simulated tpm device (assuming it is a test device with a programmatically generated ek/ srk)?

Reply:
It likely isn't relevant for TPM simulators. The TPM simulator that we use has no EK or SRK on startup. Whether we attempt to clear them here or not should result in nothing done. With that said, we may as well remove this unnecessary code